docs/sources/features/datasources/elasticsearch.md

```diff
@@ -203,6 +203,8 @@ Querying and displaying log data from Elasticsearch is available via [Explore]({
 
 Select the Elasticsearch data source, change to Logs using the Metrics/Logs switcher, and then optionally enter a lucene query into the query field to filter the log messages.
```

Comment:
It is a very long sentence. Change to:

Select the Elasticsearch data source then change to Logs using the Metrics/Logs switcher. Optionally enter a lucene query in the query field to filter log messages.